src/CssInliner.php

```diff
@@ -78,7 +78,11 @@ class CssInliner extends AbstractHtmlProcessor
     private $allowedMediaTypes = ['all' => true, 'screen' => true, 'print' => true];
 
     /**
-     * @var array<int, array<string, mixed>>
+     * @var array{
+     *         0: array<string, int>,
+     *         1: array<string, array<string, string>>,
+     *         2: array<string, string>
```

Comment:
I assume it would be asking too much to use the constant names in the type annotation.